diff --git a/src/main/java/com/sigad/sigad/business/Tienda.java b/src/main/java/com/sigad/sigad/business/Tienda.java
index 037732de..29a1ff56 100644
--- a/src/main/java/com/sigad/sigad/business/Tienda.java
+++ b/src/main/java/com/sigad/sigad/business/Tienda.java
@@ -45,7 +45,7 @@ public class Tienda {
     private Set<Usuario> empleadosTienda = new HashSet<Usuario>();
     @OneToMany(mappedBy = "tienda")
     private Set<LoteInsumo> lotesInsumo = new HashSet<LoteInsumo>();
-    @OneToMany
+    @OneToMany(mappedBy="tienda")
     private List<Vehiculo> vehiculos;
 
     /**
diff --git a/src/main/java/com/sigad/sigad/business/Vehiculo.java b/src/main/java/com/sigad/sigad/business/Vehiculo.java
index b2f15e0b..f73f13e1 100644
--- a/src/main/java/com/sigad/sigad/business/Vehiculo.java
+++ b/src/main/java/com/sigad/sigad/business/Vehiculo.java
@@ -32,6 +32,7 @@ public class Vehiculo {
         private String marca;
         private String modelo;
 
+
         public Tipo() {
         }
 
@@ -148,6 +149,8 @@ public class Vehiculo {
     private String placa;
     private String nombre;
     private String descripcion;
+    @ManyToOne
+    private Tienda tienda;
 
     public Vehiculo() {
     }
@@ -226,4 +229,18 @@ public class Vehiculo {
     public void setNombre(String nombre) {
         this.nombre = nombre;
     }
+
+    /**
+     * @return the tienda
+     */
+    public Tienda getTienda() {
+        return tienda;
+    }
+
+    /**
+     * @param tienda the tienda to set
+     */
+    public void setTienda(Tienda tienda) {
+        this.tienda = tienda;
+    }
 }
